Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -446,13 +446,18 @@ PatchAppleXcpmExtraMsrs (
 
   //
   // Now patch writes to MSR_MISC_PWR_MGMT.
+  // On macOS Sqeuoia (15) and above, this no longer exists.
   //
-  Status = PatcherApplyGenericPatch (Patcher, &mMiscPwrMgmtRelPatch);
-  if (EFI_ERROR (Status)) {
-    DEBUG ((DEBUG_INFO, "OCAK: Failed to patch writes to MSR_MISC_PWR_MGMT - %r, trying dbg\n", Status));
-    Status = PatcherApplyGenericPatch (Patcher, &mMiscPwrMgmtDbgPatch);
+  if (OcMatchDarwinVersion (KernelVersion, KERNEL_VERSION_SEQUOIA_MIN, 0)) {
+    DEBUG ((DEBUG_INFO, "OCAK: Skipping XcpmExtraMsrs MSR_MISC_PWR_MGMT patch on %u\n", KernelVersion));
```

Comment:
Hrm, how should it work then?

Reply:
Opened an issue https://github.com/acidanthera/bugtracker/issues/2410.